### ISSUE TITLE
Wrong warnings on `font-family` property in `@​font-face` directive

### ISSUE THREAD

**Issue by hail2u** [brigade/scss-lint#497]
Some fonts have a number in its name. But SCSS-Lint warns wrongly if these fonts are used in `@​font-face` directive like this:

```scss
@font-face {
  font-family: "A 1a";
}
```

```bash
$ scss-lint t.scss
t.scss:2 [W] PropertyUnits: a units not allowed on `font-family`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)
```

This happens only inside `@​font-face` directive.

**Comment by lencioni** [comment]
Thanks for the report! In my tests, this seems to be affecting all quoted values that are similar to what you included in your example, not just quoted values in `@font-face` directives. I'll push a PR for a fix in a minute.
